model/src/channel/message/mod.rs

```diff
@@ -101,23 +115,44 @@ pub struct Message {
     ///
     /// [Message Content Intent]: crate::gateway::Intents::MESSAGE_CONTENT
     pub embeds: Vec<Embed>,
+    /// Enabled [`MessageFlags`].
```

Comment:
a style decision could be made here for the whole project, maybe we shouldn't need to link to a type if the type is already in the function signature.